Resources/Prototypes/_RMC14/Entities/Objects/Weapons/specialist_loadouts.yml

```
@@ -94,9 +93,7 @@
     - id: RMCHandsM3G4
     - id: RMCArmorM3G4
     - id: RMCArmorHelmetM3G4 # hugger protection
-    - id: CMWeaponPistolMK80
-    - id: CMMagazinePistolMK80
-    - id: CMMagazinePistolMK80
+    - id: RMCMK80Case
```

This one is actually supposed to be the three items individually
```suggestion
    - id: CMWeaponPistolMK80
    - id: CMMagazinePistolMK80
    - id: CMMagazinePistolMK80
```